assets/Product.wxs

```diff
@@ -198,8 +198,8 @@
       </Directory>
     </Directory>
     <InstallExecuteSequence>
-      <Custom Action="SaveShortcutPath" After="InstallFinalize" />
-      <Custom Action="FixShortcutWorkingDirectory" After="SaveShortcutPath">
+      <Custom Action="SaveShortcutPath" Before="FixShortcutWorkingDirectory" />
+      <Custom Action="FixShortcutWorkingDirectory" Before="InstallFinalize">
```

Comment:
I find this style hard to read/maintain. The only mistake that you had before was that you ran it after `InstallFinalize`. It should be OK to run it e.g. just after `InstallFiles` and would make it more readable:
<Custom Action="SaveShortcutPath" After="InstallFiles" />
<Custom Action="FixShortcutWorkingDirectory" After="SaveShortcutPath">